server/src/main/java/io/spine/server/VisibilityGuard.java

```diff
@@ -71,7 +72,7 @@ private void checkNotAlreadyRegistered(Class<? extends Message> stateClass) {
         if (alreadyRegistered != null) {
             throw newIllegalStateException(
                     "A repository for the state class %s already registered: %s",
```

Comment:
Please add backticks for the code related parts of the message and the period at the end of the sentence.